lib/Besra/ArgParser.hs

```diff
@@ -63,7 +63,7 @@ fmtParser = Fmt <$> fmtParser' where
     (long "stdin" <> help "Whether input should be read from standard input.")
   outputMode = checkMode' <|> writeMode
   checkMode' = CheckMode <$ checkMode
-  checkMode = flag NoCheck DoCheck
+  checkMode = flag' DoCheck
```

Comment:
I think this fixes 1 scenario, but breaks another. (FromStdIn + NoCheck combination)

Reply:
@hanskenis Yup, confirmed it by adding my latest commit. If you uncomment https://github.com/luc-tielen/besra-lang/pull/37/commits/7cf636ed11d3e25675f38b20dae6f1cdcb0c57ba#diff-e3a5be69cc1aaaa8eda72253a9b17bc1R44 it will fail because of your change (but you fixed https://github.com/luc-tielen/besra-lang/pull/37/commits/7cf636ed11d3e25675f38b20dae6f1cdcb0c57ba#diff-e3a5be69cc1aaaa8eda72253a9b17bc1R34)